index.html

```diff
@@ -0,0 +1,2054 @@
+                  <h6 class="dif font-weight-small">TRAVEL</h6>
+                </div>
+                <img
+                  src="https://d.newsweek.com/en/full/1635019/san-marino.webp?w=397&h=397&f=abca48a2f88128451a6b8ca8682b20ef"
+                  alt="Top Image">
+              </div>
+              <div class="pt-3"><a class="text-decoration-none font-weight-bold dif" href="#">The Countries With the
+                  Least Amount of Travel Buzz, According to Google
+                </a></div>
+            </div>
+            <div class="d-none d-md-block col-4">
+              <div class="top_img_m position-relative">
+                <div class="position-absolute pos">
+                  <h6 class="dif font-weight-smaller">INDIA</h6>
+                </div>
+                <img
+                  src="https://d.newsweek.com/en/full/1635091/roshni-misbah-4.webp?w=397&h=397&f=9bebf36aa1f1b932999d85ad56f68a4c"
+                  alt="Top Image">
+              </div>
+              <div class="pt-3"><a class="text-decoration-none font-weight-bold dif" href="#">Delhi's 'Hijabi Biker'
+                  Breaks India's Stereotypes Riding 'Ninja' Motorbike
+                </a></div>
+            </div>
+            <div class="d-none d-md-block col-4">
+              <div class="top_img_m position-relative">
+                <div class="position-absolute pos">
+                  <h6 class="dif font-weight-small">MUSEUMS</h6>
+                </div>
+                <img
+                  src="https://d.newsweek.com/en/full/1629519/met.webp?w=397&h=397&f=4264938aed771554d52d21b49e161b24"
+                  alt="Top Image">
+              </div>
+              <div class="pt-3"><a class="text-decoration-none dif font-weight-bold" href="#">Planning a Trip to One of
+                  NYC's Recently Opened Museums? What to Know
+                </a></div>
+            </div>
+            <!--end-->
+            <!--More stories-->
+
+            <div class="row">
+              <div class="title border-bottom d-none d-md-block col-12">
+                <h4 class="p-3 text-gray font">MORE STORIES</h4>
+              </div>
+              <div class="d-none d-md-block col-12">
+                <div class="float-left p-3 pb-0"><img
+                    src="https://d.newsweek.com/en/full/1636261/facebook-ceo-mark-zuckerberg.jpg?w=95&h=95&f=89d84004bf7bf177238ad44ce9dff2e6"
+                    alt=""></div>
+                <div class="pt-3"><a class="text-decoration-none text-dark font-weight-bold" href="#">U.S. Response to
+                    Coronavirus Pandemic Rates Poorly <p>in International Poll</p></a></div>
+                <div class="pl-3 border-bottom pb-2 dif2"><span>Respondents in South Korea and Denmark were particularly
+                    harsh, with 93 percent and 92 percent, respectively, saying the U.S. has done a bad job at handling
+                    the pandemic.</span></div>
+              </div>
```

Comment:
Representing each of the article items with the `article` semantic tag would not just make your codebase look good but greatly improve SEO optimization on your site.

I encourage you to use more semantic tags instead of just div tags. It helps in debugging also. This [article](https://www.w3schools.com/html/html5_semantic_elements.asp) might be helpful to you.